internal/relationtuple/transact_server.go

```diff
@@ -52,6 +52,23 @@ func (h *handler) TransactRelationTuples(ctx context.Context, req *acl.TransactR
 	}, nil
 }
 
+func (h *handler) DeleteRelationTuples(ctx context.Context, req *acl.DeleteRelationTuplesRequest) (*acl.DeleteRelationTuplesResponse, error) {
+	if req.Query == nil {
+		return nil, errors.New("invalid request")
```

Comment:
Always return `errors.WithStack(herodot.ErrInvalidRequest.WithReason(...))` or similar errors, they will automatically set the right response codes through the middle-ware.